tests/src/test/java/alluxio/client/fs/UfsFallbackFileOutStreamIntegrationTest.java

```diff
@@ -45,20 +53,50 @@
  * persist.
  */
 @RunWith(Parameterized.class)
-public class UfsFallbackFileOutStreamIntegrationTest extends AbstractFileOutStreamIntegrationTest {
+public class UfsFallbackFileOutStreamIntegrationTest {
+  private static final int WORKER_MEMORY_SIZE = 1500;
+  private static final int BUFFER_BYTES = 100;
+
   @ClassRule
   public static ManuallyScheduleHeartbeat sManuallyScheduleEviction =
       new ManuallyScheduleHeartbeat(HeartbeatContext.WORKER_SPACE_RESERVER);
 
-  protected static final int WORKER_MEMORY_SIZE = 1500;
-  protected static final int BUFFER_BYTES = 100;
+  public static LocalAlluxioClusterResource sLocalAlluxioClusterResource =
+      new LocalAlluxioClusterResource.Builder()
+          .setProperty(PropertyKey.WORKER_MEMORY_SIZE, WORKER_MEMORY_SIZE)
+          .setProperty(PropertyKey.MASTER_PERSISTENCE_SCHEDULER_INTERVAL_MS, "100ms")
```

Comment:
can these be lower, and if so, does that speed anything up?

Reply:
I can try making them lower

Reply:
changed to 10ms